covid_xprize/examples/predictors/lstm/data/brazil_populations.csv

```diff
@@ -0,0 +1,28 @@
+GeoID,Population
```

Comment:
Is there a way to indicate the source of the data? Pandas is okay with comments in CSV files, right?

Reply:
I don't think CSV files have a standard way of specifying comments...

Reply:
Maybe a little readme in the this `data` dir?